server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -75,6 +76,7 @@ public NodesStatsRequest(StreamInput in) throws IOException {
      */
     public NodesStatsRequest(String... nodesIds) {
         super(nodesIds);
```

Comment:
could we just add a constructor to the BaseNodesRequest (concreteNodes, includeDiscoveryNodes)